### PR TITLE
Enqueue addIceCandidate

### PR DESCRIPTION
**Pull request by ekr**
ISTM that addIceCandidate needs to be queued because it has to be ordered wrt setRD.

### CODE REVIEW

**Comment by alvestrand** [comment]
This has Travis errors, but they seem unrelated to the PR.

**Comment by stefhak** [comment]
I think Ekr is right, it should be queued.

**Comment by adam-be** [comment]
This is needed. addIceCandidate() needs the remote description set so (possibly) waiting behind setRemoteDescription() in the queue makes sense.

The change is suspiciously tiny, but I believe adding the method name to the list of queued operation is sufficient as it's currently defined.

**Comment by jan-ivar** [comment]
agree

**Comment by fluffy** [comment]
LGTM

**Comment by juberti** [comment]
Convinced by Adam's argument

File: webrtc.html [hunk 1]
```diff
@@ -458,7 +458,8 @@
 
         <p>Once the RTCPeerConnection object has been initialized, for every
         call to <code>createOffer</code>, <code>setLocalDescription</code>,
-        <code>createAnswer</code> and <code>setRemoteDescription</code>;
+        <code>createAnswer</code>, <code>setRemoteDescription</code>,
+        and <code>addIceCandidate</code>;
```
Comment (jan-ivar) [inline webrtc.html]: Do we also need to add it below where it says:
> The general idea is to have only one among `createOffer`, `setLocalDescription`, `createAnswer` and `setRemoteDescription` executing at any given time.

?

**Pull request merged**